health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/presentation/component/TimeText.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.exercisesamplecompose.presentation.component
+
+import androidx.compose.runtime.Composable
+import androidx.wear.compose.material.TimeText
+import androidx.wear.compose.material.TimeTextDefaults
+
+@Composable
+fun TopOfScreenTime(){
+    TimeText(timeSource = TimeTextDefaults.timeSource(TimeTextDefaults.timeFormat()))
```

Comment:
These are the defaults in any case I think? So I think you can just use TimeText() directly?

Reply:
Done! Thanks